extensions/devservices/common/src/main/java/io/quarkus/devservices/common/ContainerAddress.java

```diff
@@ -1,23 +1,27 @@
 package io.quarkus.devservices.common;
 
+import java.util.Objects;
+
 public class ContainerAddress {
-    private final String ipAddress;
+    private final String host;
     private final int port;
 
-    public ContainerAddress(String ipAddress, Integer port) {
-        this.ipAddress = ipAddress;
+    public ContainerAddress(String host, Integer port) {
+        Objects.requireNonNull(host, "Host shouldn't be null");
+        Objects.requireNonNull(port, "Port shouldn't be null");
```

Comment:
Maybe that's just me, but the best signalization that `null` is forbidden would be to not accept `Integer`, but simply `int`.

Reply:
Yes, this is also an option - but this will require a check before passing the value to the ctor.
I can refactor it - it's not a problem.